docs/components/best-practices/modeling/choosing-the-resource-binding-type.md

```diff
@@ -21,13 +21,52 @@ For example, for a call activity this would be the version of the called process
-| :----------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `latest`     | <p>Resolves to the **latest deployed version** of the target resource at the moment the process element is activated.</p><p><ul><li><p>This can lead to unexpected behavior if you deploy a new version of the target resource without ensuring backwards compatibility with every deployed process that depends on it.</p></li><li><p>Therefore, using `latest` might not be suited for production environments that require stability and a predictable behavior.</p></li></ul></p>                                                                                                                                                                                                                                                                                                                                                                           |
-| `deployment` | <p>Resolves to the specific version of the target resource that was **deployed together** with the currently running version of the process in the **same deployment**.</p><p><ul><li><p>This option ensures predictable behavior by tying the two versions together, and allows you to deploy future versions of the target resource without disrupting ongoing process instances.</p></li><li><p>This option is ideal for self-contained projects without external or shared dependencies.</p></li><li><p>To use the `deployment` binding option, create and deploy a [process application in Web Modeler](/docs/components/modeler/web-modeler/process-applications.md#deploy-and-run-a-process-application), or deploy multiple resources together via the [Zeebe API](/docs/apis-tools/zeebe-api/gateway-service.md#deployresource-rpc).</p></li></ul></p> |
+<table>
```

Comment:
FYI: I've converted the table from Markdown format to HTML to make the code easier to read and maintain.

Reply:
💭 Would be helpful to split this conversion from the textual changes into different commits, to more easily see the differences in the text itself ([separate structural from behavioral changes](https://github.com/camunda/camunda/wiki/Pull-Requests-and-Code-Reviews#ive-separated-structural-from-behavioral-changes)).

Reply:
Ah, yes, sorry about that! I've split the commit into two now (at least for the technical writer review 😉), but they'll be squashed again on merge.